src/components/WorkOrder/WorkOrderDetails.test.js

```diff
@@ -102,6 +102,30 @@ describe('WorkOrderDetails component', () => {
       )
       expect(asFragment()).toMatchSnapshot()
     })
+    it('should not render Multibutton if work order is closed', () => {
```

Comment:
This is a bit of nitpick but it's a best practice IMO to avoid referring only to generic names for component or other generic units of code in test descriptions. It's unlikely I admit, but we could in the future add an additional multibutton, for some other functionality, which is always visible. That would then make this test description incorrect and might confuse the reader. Alternatively, we might replace the existing multibutton in the future with another implementation like a set of separate action buttons and this test description could lead to confusion unless we remember to update it.

I suggest we call it something more specific to repairs hub like 'should not render the work order action menu ...'

Reply:
Or 'should not render the work order action multibutton...' would also be fine. It's just to avoid referencing just the generic component on its own. Nitpick essay over!